content/ember-4-0-released.md

```diff
@@ -0,0 +1,240 @@
+
+Read more about this change in the [deprecation guide](https://deprecations.emberjs.com/v3.x/#toc_3-0-browser-support-policy) and at Ember's [browser support policy page](https://emberjs.com/browser-support/).
+
+### `ember-auto-import` is now required
+
+Using Ember 4 or higher requires that the project has version `2.0.0` or later
+of `ember-auto-import` installed. Follow the "How to upgrade" steps above to
+learn what you need to do.
+
+Why is this change required? `ember-auto-import` supports something we call the
+[v2 addon
+format](https://emberjs.github.io/rfcs/0507-embroider-v2-package-format.html).
+This specification describes how to publish an Ember CLI addon that has minimal
+build overhead and better compatibility with modern build and analysis tooling
+(think [Webpack](https://webpack.js.org/)). Later in Ember's 4.x release cycle,
+we intend to release framework libraries as v2 addons to take advantage of those
+improvements.
+
+## Changes in Ember.js 4.0
+
+Ember.js is the core of the Ember framework. It provides routing, rendering, and
+dependency injection features.
+
+Ember.js 4.0 introduces no new public API. Instead, it is comprised of bug fixes
+and the removal of previously deprecated public API from the 3.x cycle.
+
+Ember 4.0 does _not_ remove the `EmberComponent` API or the core parts of the
+`EmberObject` system. These APIs are widely used, even after the release of
+Octane, by existing application and addon code.
+
+### APIs Removed in 4.0
+
+Below we've listed some of the most significant API removals in Ember.js 4.0.
+For a complete list of removals, see the [Ember.js 3.x deprecation
+guide](https://deprecations.emberjs.com/v3.x).
+
+- `Ember.Logger` is removed in favor of native `console` APIs. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_ember-console-deprecate-logger).
+- `Copyable` mixin is removed in favor of the [ember-copy addon](https://github.com/emberjs/ember-copy). [Guide here](https://deprecations.emberjs.com/v3.x/#toc_ember-runtime-deprecate-copy-copyable).
+- `sendAction` is removed in favor of calling closure actions like any other callback. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_ember-component-send-action).
+- `willTransition` and `didTransition` are removed in favor of router service events. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_deprecate-router-events).
+- Computed Property `volatile()` calls are removed in favor of native getters. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_computed-property-volatile).
+- `this.$()` and other jQuery APIs are deprecated in favor of native browser equivalents. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_jquery-apis). An optional feature which restored this and other jQuery-specific features is also removed. [Guide here](https://deprecations.emberjs.com/v3.x/#toc_optional-feature-jquery-integration).
```

Comment:
Is this deprecated, or removed in v4? This bullet point should be updated if it has been removed.

Reply:
Looking at [this PR](https://github.com/emberjs/rfc-tracking/issues/3) I don't think the work was completed, but maybe @rwjblue or @simonihmig can check the list to see if the work was completed and the list just wasn't updated.

Reply:
The list wasn't updated, will leave a comment there. The work is done. And yes, they have been *removed*!

Reply:
I've updated the text here appropriately. I expect it was left over from a copy-paste job.